src/records.ts

```diff
@@ -230,6 +230,12 @@ export interface TraitsRecord {
    * web service or the Enterprise database.
    */
   readonly autonomousSystemOrganization?: string;
+  /**
+   * The connection type may take the following values:
+   * "Dialup", "Cable/DSL", "Corporate", "Cellular".
+   * Additional values may be added in the future.
+   */
+  readonly connectionType?: string;
```

Comment:
Since the type values are known or enum, rather than just a string?

Reply:
Would it be okay to use a union type instead of an enum?  The former is a lot simpler and I think it fits the use-case a bit better.

Reply:
Damn. I butchered my sentence in my previous reply. Kudos for deciphering that.

Yeah, I think a union would be good here.

Reply:
No worries.  I've pushed the change.